src/model/article-history.v1.yaml

```diff
@@ -1,25 +1,42 @@
 $schema: http://json-schema.org/draft-04/schema#
-title: Article history
-type: object
-properties:
-    received:
-        type: string
-        format: date
-    accepted:
-        type: string
-        format: date
-    versions:
-        description: List of article versions, sorted by version in descending order
-        type: array
-        items:
-          - oneOf:
-              - $ref: ../snippets/article-poa.v1.yaml
-              - $ref: ../snippets/article-vor.v1.yaml
-        minItems: 1
-dependencies:
-    received:
-      - accepted
-    accepted:
-      - received
-required:
-    - versions
+anyOf:
+-
+    title: Article history
+    type: object
+    properties:
+        received:
+            type: string
+            format: date
+        accepted:
+            type: string
+            format: date
+        versions:
+            description: List of article versions, sorted by version in descending order
+            type: array
+            items:
+              - oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+    dependencies:
+        received:
+          - accepted
+        accepted:
+          - received
+    required:
+        - versions
+-
+    title: Publication Event
```

Comment:
this is more in line with what I meant, not tying this to the other data type of "marketing" events